x-pack/plugins/reporting/server/routes/jobs.ts

```diff
@@ -7,6 +7,7 @@
 
 import { schema } from '@kbn/config-schema';
 import Boom from '@hapi/boom';
+import { ROUTE_TAG_CAN_REDIRECT } from '../../../security/server';
```

Comment:
__note:__ I was going back and forth on this one, and it feels like using route tags is the safest and easiest approach until we get support from the Core (e.g route `canRedirect: boolean` option) which will likely be very similar to the tags anyway.